src/netius/test/servers/ftp.py

```
@@ -113,3 +122,450 @@ def test_on_line_method(self):
         # connection is never able to reach it
         for code in ("LINE", "FLUSH_LIST", "FLUSH_RETR"):
             self.assertRaises(netius.ParserError, self.connection.on_line, code, "")
+
+
+class FTPConnectionPathTest(unittest.TestCase):
+
+    def setUp(self):
+        unittest.TestCase.setUp(self)
+        if mock == None:
```

Ruff (0.16.3)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)